### ISSUE TITLE
(Huawei Mediapad M5 SHT-W09) apps can't access /data/user/0/

### ISSUE THREAD

**Issue by darren3** [phhusson/treble_experimentations#37]
Any apps I install seem unable to access /data/user/0/. Apps are crashing on open or not remembering settings. I have experienced this on versions 17 and 18 of Phh-Treble AOSP (Gapps and SU). I have tried factory resets and erasing userdata via fastboot but it hasn't helped. I have attached a logcat, would you be able to look at it for me? 

[mediapad.txt](https://github.com/phhusson/treble_experimentations/files/2011031/mediapad.txt)

**Comment by Minifiles** [comment]
Try to flash the exact same rom again and wipe the data. This worked for me on my Huawei P Smart

**Comment by darren3** [comment]
I managed to sort my data partion out. I rebooted to bootloader, wiped data with fastboot -w, wiped system with fastboot erase system, flashed the system image, wiped data again with fastboot -w and then booted straight to recovery and formated the data. Previously I hadn't booted straight to recovery and had a couple of bootloops and the device would go to recovery and ask for format. I love this ROM I bought this tablet (to replace my Nexus 7) of the back of it being highly specced and supporting project treble knowing I could use this ROM.
